packages/worker/src/api/routes/index.ts

```diff
@@ -39,4 +39,5 @@ export const routes: Router[] = [
   accountRoutes,
   restoreRoutes,
   eventRoutes,
+  ...api.scimRoutes,
```

Comment:
Not directly related to these changes but could we update the naming here to:
```ts
import { api as pro } from "@budibase/pro"
```

Also looks like we pull the `userGroupRoutes` and `auditLogRoutes` out into their own variables. Could we be consistent with the `scimRoutes` and either
- use a variable, or
- use the spread pattern for the others too